reframe/frontend/cli.py

```diff
@@ -1021,6 +1021,18 @@ def module_unuse(*paths):
                     f'failed to generate report in {report_file!r}: {e}'
                 )
 
+            # Build xml report
+            xml_data = runner.stats.junit(json_report).decode()
+            xml_report_file = os.path.splitext(report_file)[0]+'.xml'
```

Comment:
@teojgo would this be possible (if yes, how) ?
- `--junit-xml` then xml_rpt_file=`'rpt_file'.xml`
- `--junit-xml=0.xml`  then xml_rpt_file=`0.xml`
- no `--junit-xml` then no xml_rpt_file

Reply:
I have some comment about the options. I am not entirely in favour of these options as such. I'll get back to it later.

Reply:
Why not something like `--report-junit[=FILE]` very similar to what we do with the `--report-file`? We should also support the exact same syntax in this case, as we do with the report file. Check [here](https://reframe-hpc.readthedocs.io/en/stable/manpage.html?highlight=--report-file#cmdoption-report-file).

Reply:
https://github.com/eth-cscs/reframe/pull/1925/commits/286cc91447a985e497ced5e53fbde52ef147e975